package.json

```diff
@@ -59,6 +59,7 @@
     "@types/redux-mock-store": "^1.0.6",
     "@typescript-eslint/eslint-plugin": "^5.59.0",
     "@typescript-eslint/parser": "^5.59.0",
+    "autoprefixer": "^10.4.20",
```

Comment:
c'mon 2024. Are we really still need it?

Reply:
resolved

Reply:
resolved